### DISCUSSION TITLE
HACS custom integration component

### DISCUSSION THREAD

**Discussion by menaibluey** [PeterH24x7/-Solar-Analytics-integration-for-Home-Assistant-Energy-monitoring#21] [Ideas]
I was wondering whether you ever considered creating a HACS custom integration component for this with configuration exposed via UI? Not sure how involved that would be? Found some templates at https://github.com/oncleben31/cookiecutter-homeassistant-custom-component

**Comment by PeterH24x7** [comment]
Hi @menaibluey - thank you for the suggestion. I gather in the absence of the HACS integration that you otherwise have managed to get Solar Analytics working with the code as provided? Others have made a similar suggestion and I have looked into it in the past but it didn't seem to be worth the effort where those on this forum already have it working. I'd be happy for someone to assist, but simply don't have the time at this stage.
Chrs, Peter

**Comment by PeterH24x7** [comment]
Nothing further to discuss here I think. Will close the discussion.
